config/custom_items.txt

```diff
@@ -936,6 +936,12 @@ character_name: Zeke Arachi
 item_path: /obj/item/clothing/head/fluff/xeno
 }
 
+{
+ckey: Zigfe
```

Comment:
will not work, must be lowercase in 'canonized' aka ckey format

Reply:
I feel like I've been told this same thing before, when I made my own fluff item PR